### PR TITLE
[Feature] add envs FD_USE_PHI_FP8_QUANT

### PR DESCRIPTION
**Pull request by fxyfxy777**
## Motivation

> :bulb: If this PR is a Cherry Pick, the PR title needs to follow the format by adding the [Cherry-Pick] label at the very beginning and appending the original PR ID at the end. For example, [Cherry-Pick][CI] Add check trigger and logic(#5191)

> :bulb: 如若此PR是Cherry Pick，PR标题需遵循格式，在最开始加上[Cherry-Pick]标签，以及最后面加上原PR ID，例如[Cherry-Pick][CI] Add check trigger and logic(#5191)

## Modifications
FD_USE_PHI_FP8_QUANT
控制是否使用fd的量化算子，默认为1，使用paddle的量化算子

## Checklist

- [ ] Add at least a tag in the PR title.
  - Tag list: [`[FDConfig]`,`[APIServer]`,`[Engine]`, `[Scheduler]`, `[PD Disaggregation]`, `[Executor]`, `[Graph Optimization]`, `[Speculative Decoding]`, `[RL]`, `[Models]`, `[Quantization]`, `[Loader]`, `[OP]`, `[KVCache]`, `[DataProcessor]`, `[BugFix]`, `[Docs]`, `[CI]`, `[Optimization]`, `[Feature]`, `[Benchmark]`, `[Others]`, `[XPU]`, `[HPU]`, `[GCU]`, `[DCU]`, `[Iluvatar]`, `[Metax]`]
  - You can add new tags based on the PR content, but the semantics must be clear.
- [ ] Format your code, run `pre-commit` before commit.
- [ ] Add unit tests. Please write the reason in this PR if no unit tests.
- [ ] Provide accuracy results.
- [ ] If the current PR is submitting to the `release` branch, make sure the PR has been submitted to the `develop` branch, then cherry-pick it to the `release` branch with the `[Cherry-Pick]` PR tag.

### CODE REVIEW

File: fastdeploy/envs.py [hunk 1]
```diff
@@ -164,6 +164,8 @@
     "GLOBAL_LOGGING_INSTRUMENT": lambda: int(os.getenv("GLOBAL_LOGGING_INSTRUMENT", "0")),
     # Timeout for worker process health check in seconds
     "FD_WORKER_ALIVE_TIMEOUT": lambda: int(os.getenv("FD_WORKER_ALIVE_TIMEOUT", "30")),
+    # Whether to use phi FP8 quantization,if 1,use paddle default.
+    "FD_USE_PHI_FP8_QUANT": lambda: int(os.getenv("FD_USE_PHI_FP8_QUANT", "1")),
```
Comment (yuanlehome) [inline fastdeploy/envs.py]:
```suggestion
    "FD_USE_PHI_FP8_QUANT": lambda: bool(int(os.getenv("FD_USE_PHI_FP8_QUANT", "1"))),
```

File: fastdeploy/model_executor/layers/quantization/block_wise_fp8.py [hunk 2]
```diff
@@ -225,10 +226,15 @@ def process_prequanted_weights(self, layer, state_dict, is_rearrange: bool = Fal
         layer.weight_scale_inv.set_value(weight_scale)
 
     def apply(self, layer, x):
-        x, x_scale_tensor = paddle.incubate.nn.functional.fp8_quant_blockwise(
-            x, using_pow2_scale=False, output_scale_transpose=True
-        )
-        x_scale_tensor = x_scale_tensor.T
+        if fastdeploy.envs.FD_USE_PHI_FP8_QUANT == 0:
```
Comment (yuanlehome) [inline fastdeploy/model_executor/layers/quantization/block_wise_fp8.py]:
```suggestion
        if not fastdeploy.envs.FD_USE_PHI_FP8_QUANT:
```

Comment (fxyfxy777) [reply fastdeploy/model_executor/layers/quantization/block_wise_fp8.py]: update

File: fastdeploy/envs.py (hunk 1 above)
Comment (fxyfxy777) [reply fastdeploy/envs.py]: update

**Review by yuanlehome — APPROVED** [review]
